src/ssl.c

```diff
@@ -40568,6 +41003,38 @@ int wolfSSL_i2d_X509_NAME(WOLFSSL_X509_NAME* name, unsigned char** out)
         return (WOLFSSL_X509* )wolfSSL_PEM_read_X509_ex(fp, (void **)x, cb, u, CERT_TYPE);
     }
 
+#ifndef NO_BIO
+    WOLFSSL_EVP_PKEY* wolfSSL_PEM_read_PrivateKey(XFILE fp,
+        WOLFSSL_EVP_PKEY **x, pem_password_cb *cb, void *u)
+    {
+        int err = 0;
+        WOLFSSL_EVP_PKEY* ret = NULL;
+        WOLFSSL_BIO* bio = NULL;
+
+        WOLFSSL_ENTER("wolfSSL_PEM_read_PrivateKey");
+
+        if (fp == NULL) {
```

Comment:
This check should be `fp == XBADFILE`. Otherwise may fail comparison on some ports.